### PR TITLE
fix ansible lint errors

### PR DESCRIPTION
**Pull request by johanneskastl**
Only errors left now:

```bash
WARNING  Listing 3 violation(s) that are fatal
no-changed-when: Commands should not change things if nothing needs doing.
playbook.yml:63 Task/Handler: Create the podman secret

no-changed-when: Commands should not change things if nothing needs doing.
playbook.yml:75 Task/Handler: Create the kube secret

no-changed-when: Commands should not change things if nothing needs doing.
playbook.yml:79 Task/Handler: Create the podman secret
```

I'll see if I can fix them tomorrow.

### CODE REVIEW

File: quadlet-files/quadlet-demo.yml [hunk 1]
```diff
@@ -18,50 +18,50 @@ metadata:
...
-      requests:
-        memory: "64Mi"
-        cpu: "250m"
-      limits:
-        memory: "128Mi"
-        cpu: "500m"
-  - name: envoy
-    image: docker.io/envoyproxy/envoy:v1.25.0
-    volumeMounts:
+    - name: wordpress
```
Comment (ygalblum) [inline quadlet-files/quadlet-demo.yml:21]: I don't like over indentation of arrays (you can see that the playbook does not do that)

**Review by ygalblum — CHANGES_REQUESTED** [review]
Thanks for the PR. 
Please note that you need to rebase (following your other changes)

File: quadlet-files/quadlet-demo.yml (hunk 1 above)
Comment (johanneskastl) [reply quadlet-files/quadlet-demo.yml:21]: Those are changes that ansible-lint aka yamllint proposed. For $REASONS it does not complain for playbooks not indenting like this.

But I can of course just drop the commit, if you wish.

**Comment by johanneskastl** [comment]
> Thanks for the PR. Please note that you need to rebase (following your other changes)

Thank you for the nice demo setup. And thanks for taking time to review.

I'll rebase this later, no problem.

File: quadlet-files/quadlet-demo.yml (hunk 1 above)
Comment (ygalblum) [reply quadlet-files/quadlet-demo.yml:21]: I've added a .yamllint file that aligns with my convention and should remove this error. Just rebase

Comment (johanneskastl) [reply quadlet-files/quadlet-demo.yml:21]: > I've added a .yamllint file that aligns with my convention and should remove this error. Just rebase

Thanks, but then the indentation in the playbook is off. I can of course fix that, too. But then it does not look like Ansible playbooks normally do...

Comment (ygalblum) [reply quadlet-files/quadlet-demo.yml:21]: What command are you running? When I run `ansible-lint playbook.yml` the only indentation error I'm getting is about your addition of the Firewall rules

Comment (johanneskastl) [reply quadlet-files/quadlet-demo.yml:21]:
```
$ ansible-lint --offline --profile=production
```

Might be a difference in the version of ansible-lint and yamllint:
```
$ ansible-lint --version
ansible-lint 6.14.2 using ansible 2.14.3
$ yamllint --version
yamllint 1.28.0
$
```

Comment (johanneskastl) [reply quadlet-files/quadlet-demo.yml:21]: Sorry, I forgot to drop the "fix quadlet-demo.yml" commit during the rebase.

Not it looks better. Hold on...

**Comment by johanneskastl** [comment]
OK, so now it looks good without touching the kubernetes YAML file indentation.

Only the no-changed-when errors remain. I'll have a look later.

**Review by ygalblum — APPROVED** [review]

**Pull request merged**